src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends ColorSupplier implements Figure {
+    private static int side1;
+    private static int side2;
+
+    public  static int getSide1() {
+        return side1;
+    }
+
+    public static int getSide2() {
+        return side2;
+    }
+
+    public void setSide1(int side1) {
+        this.side1 = side1;
+    }
+
+    public void setSide2(int side2) {
+        this.side2 = side2;
+    }
+
+    public static int getArea(){

```

Comment:
All methods related to 'side1' and 'side2' should not be static. Static methods and fields are shared among all instances of a class, which is not the intended behavior for instance variables like the sides of a square.